shortcircuit-api/src/main/java/com/powsybl/shortcircuit/FaultResult.java

```diff
@@ -37,8 +50,25 @@ public String getId() {
     /**
      * Value of the 3-phase short-circuit current for this fault (in A).
      */
-    public float getThreePhaseFaultCurrent() {
+    public double getThreePhaseFaultCurrent() {
         return threePhaseFaultCurrent;
     }
 
+    /**
+     * List of contributions to the three phase fault current of each connectable connected to the equipment
+     */
+    public List<FeederResult> getFeederResults() {
+        return feederResults;
+    }
+
+    public double getFeederCurrent(String feederId) {
+        double current = 0;
+        for (FeederResult feederResult : feederResults) {
+            if (feederResult.getConnectableId().equals(feederId)) {
+                current = feederResult.getFeederThreePhaseCurrent();
```

Comment:
For performance purposes, I think you should return the current as soon as you find it (avoiding going through all feederResult). You should have something like this:
```java
for (FeederResult feederResult : feederResults) {
      if (feederResult.getConnectableId().equals(feederId)) {
            return feederResult.getFeederThreePhaseCurrent();
      }
}
return Double.NaN;
```